nautobot/virtualization/tests/test_filters.py

```diff
@@ -638,22 +640,40 @@ def test_mode(self):
 
     def test_ip_addresses(self):
```

Comment:
Are these changes fixing #2689?

Reply:
Here is the change I made that seems to fix the error on my end. https://github.com/nautobot/nautobot/commit/6ab61c3e795480ba76f417d7a1b7808d1784aa37

Reply:
> Are these changes fixing #2689?

They are. Thanks for the reminder